pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -508,6 +511,32 @@ func (f *framework) runFilterPlugin(ctx context.Context, pl FilterPlugin, state
 	return status
 }
 
+// RunPostFilterPlugins runs the set of configured PostFilter plugins. It will stop iterating the postScorePlugins
+// when it meets the first Schedulable status.
+// It returns an optional nominatedNodeName as well as the
+func (f *framework) RunPostFilterPlugins(ctx context.Context, state *CycleState, pod *v1.Pod, filterErr error) (string, PluginToStatus) {
+	statuses := make(PluginToStatus)
+	for _, pl := range f.postFilterPlugins {
+		nnn, status := f.runPostFilterPlugin(ctx, pl, state, pod, filterErr)
```

Comment:
same as above, `nnn` could be more descriptive as it's not obvious from `runPostFilterPlugin` that this is where the nominated node name comes from